app/src/main/kotlin/com/github/oryanmat/trellowidget/util/network/VolleyUtil.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.oryanmat.trellowidget.util.network
+
+import android.content.Context
+import com.android.volley.Request
+import com.android.volley.RequestQueue
+import com.android.volley.toolbox.RequestFuture
+import com.android.volley.toolbox.StringRequest
+import com.android.volley.toolbox.Volley
+import com.github.oryanmat.trellowidget.util.Json
+import java.lang.reflect.Type
+
+class VolleyUtil(appContext: Context) {
```

Comment:
I don't think this class makes sense. Unless we're planning on adding more APIs (I don't think so?) these are just private methods in `TrelloApi`

Reply:
It's more of a personal preference, imo. I think this separation helps keep the code organized and maintainable. But if we prefer `TrelloApi` compact, I'm fine with merging this class.